lib/components/map/point-popup.tsx

```diff
@@ -18,17 +19,18 @@ const PopupTitle = styled.div`
 `
 
 function MapPopup({
+  intl,
   mapPopupLocation,
   onSetLocationFromPopup,
   setMapZoom,
   zoom
 }: {
-  mapPopupLocation: { name: string }
+  mapPopupLocation: { lat: number; lon: number; name?: string }
```

Comment:
Use the `Place` type from `@opentripplanner/types`?

Reply:
Beautiful! Done in 6d5bc90e